src/stream/src/executor/managed_state/aggregation/value.rs

```diff
@@ -31,34 +33,40 @@ pub struct ManagedValueState<S: StateStore> {
     /// The internal single-value state.
     state: Box<dyn StreamingAggStateImpl>,
 
-    /// The keyspace to operate on.
-    keyspace: Keyspace<S>,
-
     /// Indicates whether this managed state is dirty. If this state is dirty, we cannot evict the
     /// state from memory.
     is_dirty: bool,
+
+    /// Relational table used by this Agg state. One relational table may shared by multiple agg
+    /// state.
+    state_table: Arc<Mutex<StateTable<S>>>,
```

Comment:
Wrap it with a `Mutex` seems strange since there's actually no concurrent access for now... Not sure how much it will cost. Are there any alternatives to this? For example, passing a mutable reference of the table when `flush`ing like we used to do for `Keyspace`?

Reply:
I was also thinking on this. The question is where to put the `StateTable` fields. If we put in value state and hide it from executor, then there must be a Rc or Arc stuff (`Rc<RefCell<>>` may ok here? not sure whether will be rejected by async-future-stream).

If we put it in executor fields, then executor's may aware state table and Agg State. But we may able to &mut it and pass to agg states (Just like WriteBatch)

Reply:
Seems every agg state owns a different instance of state table in this PR?

Reply:
Indeed my fault! Will fix this stupid bug..

Reply:
Fixed